src/components/NavBar.vue

```diff
@@ -10,6 +10,21 @@
       <b-navbar-nav>
         <b-nav-item :to="{ name: 'about' }">About</b-nav-item>
       </b-navbar-nav>
+      <b-form-input
+        class="search ml-auto"
+        v-model="searchString"
+        v-if="username"
+        placeholder="Search Compounds (cid or inchikey)"
+      />
+      <b-button
+        class="ml-2"
+        variant="dark"
+        @click="searchCompound"
+        v-if="username"
+        :disabled="searchString.length == 0"
+      >
+        <b-icon icon="search" />
+      </b-button>
```

Comment:
This search bar gets mangled when resizing to the compact (mobile) navbar view.

Reply:
I've resolved this by putting the search bar and button into it's own `<b-navbar-nav>` element. There will be a problem with a static size for the iframe on mobile devices, but I don't think that we are developing for screen sizes this small.